pd-configuration-file.md

```diff
@@ -104,6 +104,48 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 
 pd-server 相关配置项。
 
+### `server-memory-limit` <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
+> **警告：**
+>
+> 在当前版本中，该配置项为实验特性，不建议在生产环境中使用。
+
++ PD 实例的内存限制比例。`0` 值表示不设内存限制。
++ 默认值：`0`
++ 最小值：`0`
++ 最大值：`0.99`
+
+### `server-memory-limit-gc-trigger` <span class="version-mark">从 v6.6.0 版本开始引入</span>
+
+> **警告：**
+>
+> 在当前版本中，该配置项为实验特性，不建议在生产环境中使用。
+
++ PD 尝试触发 GC 的阈值比例。当 PD 的内存使用达到 `server-memory-limit` 值 * `server-memory-limit-gc-trigger` 值时，则会主动触发一次 Golang GC。在一分钟之内只会主动触发一次 GC。
```

Comment:
```suggestion
+ PD 尝试触发 GC 的阈值比例。当 PD 的内存使用达到 `server-memory-limit * server-memory-limit-gc-trigger` 时，则会主动触发一次 Golang GC。在一分钟之内只会主动触发一次 GC。
```

Reply:
Thanks a lot for your comments. The phrasing in these series parameters' doc is mainly referred from TiDB's doc,  please see [here](https://docs.pingcap.com/zh/tidb/stable/system-variables#tidb_server_memory_limit_gc_trigger-%E4%BB%8E-v640-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5). So I think maybe the current description is also acceptable.